test/test_crypto.py

```diff
@@ -209,47 +209,47 @@ def test_get_signer_cert(self):
         if cert.strip() != cert_string.strip():
             self.fail('Certificate retrieved from signature '
                       'does not match certificate used to sign.')
-        
+
     def test_encrypt(self):
         '''
         Not a correct test yet.
         '''
         encrypted = encrypt(MSG, TEST_CERT_FILE)
-        
+
         if not 'MIME-Version' in encrypted:
             self.fail('Encrypted message is not MIME')
-        
+
         # Indirect testing, using the decrypt_message function.
         decrypted = decrypt(encrypted, TEST_CERT_FILE, TEST_KEY_FILE)
-        
+
         if decrypted != MSG:
             self.fail("Encrypted message wasn't decrypted successfully.")
-            
+
         # invalid cipher
         try:
             encrypted = encrypt(MSG, TEST_CERT_FILE, 'aes1024')
         except CryptoException:
-            pass    
-        
-            
+            pass
+
```

Comment:
Would be nice to remove the extra line - could that be put with your tabs to spaces commit as a non-trailing whitespace fixing PR?